Mu2eG4/geom/tracker_straw_params_v01.txt

```diff
@@ -0,0 +1,25 @@
+// Mu2e era straw parameters taken from tracker_v6.txt
+
```

Comment:
These are parameters have the same values as for tracker_v6.txt.  What's the plan - verify with these parameters and then make a new file with updated parameters?

Reply:
The point of this file is to be able to switch back to the Mu2e era straws while testing the Mu2e-II era versions. I don't believe there will be any updates to this file, unless the Mu2e era straw parameters are updated during this study. A use case would be to run CE reco with and without the updated straw parameters, to see the effect on the track resolution.

Reply:
Thanks.